README.md

```diff
@@ -69,7 +86,7 @@ This section provides a general introduction on how to start using the app. You
     | PATHWAY_REST_CONNECTOR_PORT | Specifies the port number on which the REST connector service of the Pathway should listen. Here, it is set to 8080.                                                                                                                                       |
     | OPENAI_API_TOKEN            | The API token for accessing OpenAI services. If you are not running the local version, please remember to replace it with your personal API token, which you can generate from your account on [openai.com](https://platform.openai.com/account/api-keys). |
     | PATHWAY_CACHE_DIR           | Specifies the directory where cache is stored. You could use /tmp/cache.                                                                                                                                                                                   |
```

Comment:
Could you please check why this table isn't rendering correctly?

Reply:
Now it should be fine. There was an issue with positioning.